Welthungerhilfe/cgm-ml
# 964 Roll out logging in repository

# Description
Logging concept has been approved on 
- [x] CNNDepthMap/CNNDepthMap-height/q3-depthmap-plaincnn-height
and is now ready for roll out in the repository.
For a start, all files in cgm_ml/src/models will be updated.

related to #274 #148 
Component Requirement https://dev.azure.com/cgmorg/ChildGrowthMonitor/_sprints/backlog/ChildGrowthMonitor%20Team/ChildGrowthMonitor/2021.Q1/Q1.Sprint%203?workitem=964

# Model Folders to be updated
CNNDepthMap
- [x] CNNDepthMap-height/q2milestone-cnndepthmap-height
- [x] CNNDepthMap-height/q2s1-cnndepthmap.height
- [x] CNNDepthMap-height/q3-cnndepthmap-resnet-height
- [x] CNNDepthMap-height/q3-depthmapmultiartifact-plaincnn-height
- [x] CNNDepthMap-height/q3-depthmapmultiartifactlatefusion-plaincnn-height
- [x] CNNDepthMap-height/s4-cnndepthmap-height
- [x] CNNDepthMap-height/s5-cnndepthmap-height
- [x] CNNDepthMap-multitask/q4-depthmap-plaincnn-multitask
- [x] CNNDepthMap-weight/s5-cnndepthmap-weight

CNNRGB
- [x] CNNRGB-standing_laying/q4-rgb-standing-laying

GAPNet
- [x] GAPNet-height/s1-gapnet-height
- [x] GAPNet-height/s2-gapnet-height

PersonLab
- [x] PersonLabV1

pointnet
- [x] pointnet-height/pointnet-20190708
- [x] pointnet-height/pointnet-20190806

Unsupervised
- [x] Autoencoder/q4-anomalydetection-ae

# How Has This Been Tested?

Models above are only ticked if logs work on compute cluster during training.
- run train.py locally
- run training scripts with nbr of epochs = 2
- check logging output in azureml_logs/70_driver_log.txt
- in some cases, training scripts could not be run bec. model not updated after moving tenants, model not curated, etc
- since the logging statements worked in most of the files, it can be assumed that it also works in the others 

# Checklist:

- [ ] My Code is review by 1 People
